public/app/features/alerting/unified/Receivers.test.tsx

```diff
@@ -183,22 +190,37 @@ describe('Receivers', () => {
     // try to test the contact point
     userEvent.click(ui.testContactPointButton.get());
 
+    await waitFor(() => expect(ui.testContactPointModal.get()).toBeInTheDocument());
+    userEvent.click(ui.customContactPointOption.get());
+    await waitFor(() => expect(ui.contactPointAnnotationSelect(0).get()).toBeInTheDocument());
+
+    // enter custom annotations and labels
+    await clickSelectOption(ui.contactPointAnnotationSelect(0).get(), 'Description');
+    await userEvent.type(ui.contactPointAnnotationValue(0).get(), 'Test contact point');
+    await userEvent.type(ui.contactPointLabelKey(0).get(), 'foo');
+    await userEvent.type(ui.contactPointLabelValue(0).get(), 'bar');
```

Comment:
```suggestion
    await userEvent.type(ui.contactPointLabelKey(0).get(), 'foo');
    await userEvent.type(ui.contactPointLabelValue(0).get(), 'bar');
```
What do `foo` and `bar` mean here?

Feel free to disregard because this is just test code.

Reply:
Also, just spotted @achatterjee-grafana on the reviewer list. Good. :)